src/main/zapHomeFiles/hud/entrypoints/Management/management.js

```diff
@@ -0,0 +1,212 @@
+/*
+ * Management Frame
+ *
+ * Initializes the service worker and forwards messages between the service worker
+ * and inject.js.
+ */
+
+// temp time test
+// var startTime = new Date().getTime();
+//TODO: Refactor tabId as props
+let tabId = "";
+
+// // Injected strings
+const SHOW_WELCOME_SCREEN = "<<SHOW_WELCOME_SCREEN>>" === "true" ? true : false;
+const ZAP_SHARED_SECRET = process.env.ZAP_SHARED_SECRET;
+
+import Vue from "vue";
+import localforage from "localforage";
```

Comment:
Leaving this here, because I see a number of problems with the implementation of localforage - I think that for the reasons that follow you would do well to create an export similar to the one for the EventBus. That way you could configure it, and potentially extend the interface (if you are so inclined).

For one, it seems to me that you are really expecting that localstorage is a safe place to store things. It isn't. The localForage serializer does not protect, just convert to and from base64. Should a developer's  localhost (for example) get poisoned with appropriately base64'd functions, you would have no way of stopping them from reeking havoc. I think the only way to harden this interface is to actually pass it through e.g. DOMPurify. 

Furthermore, there are a number of places in the code where you aren't catching errors. What happens if I don't have any more storage space available? localForage will pass that as an error, but how will this interface react?